src/sgx/base64.cpp

```diff
@@ -0,0 +1,143 @@
+
+#include <sgx/base64.h>
+#include <string>
+
+//#include <string.h>
+    /**
+    * Base64 encoder/decoder.
+    * Modified from file found at
+    * https://github.com/gaspardpetit/base64/blob/master/src/apache/apache.h
+    */
+
+static const char basis_64[] =
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+/* aaaack but it's fast and const should make it shared text page. */
```

Comment:
The style of comments in this PR is quite inconsistent, but this is particularly uninformative, what does it mean?